moqu/deployment/src/main/java/io/quarkiverse/openapi/generator/MoquProjectProcessor.java

```diff
@@ -0,0 +1,102 @@
+package io.quarkiverse.openapi.generator;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Objects;
+import java.util.Optional;
+import java.util.Set;
+import java.util.stream.Stream;
+
+import org.jboss.logging.Logger;
+
+import io.quarkiverse.openapi.generator.items.MoquBuildItem;
+import io.quarkiverse.openapi.generator.items.MoquProjectBuildItem;
+import io.quarkiverse.openapi.generator.moqu.MoquConfig;
+import io.quarkiverse.openapi.moqu.Moqu;
+import io.quarkiverse.openapi.moqu.OpenAPIMoquImporter;
+import io.quarkus.deployment.IsDevelopment;
+import io.quarkus.deployment.annotations.BuildProducer;
+import io.quarkus.deployment.annotations.BuildStep;
+import io.quarkus.runtime.util.ClassPathUtils;
+
+public class MoquProjectProcessor {
+
+    private static final Logger LOGGER = Logger.getLogger(MoquProjectProcessor.class);
+
+    private static final Set<String> SUPPORTED_EXTENSIONS = Set.of("yaml", "yml");
```

Comment:
Not json?

Reply:
I was thinking to add in another release, but makes sense to have all, it is simple!